File: java/src/main/java/com/newnancity/newnanmanager/AuthScheme.java
package com.newnancity.newnanmanager;

/** Authentication scheme used by the NewNanManager HTTP API. */
public enum AuthScheme {
    /** Sends {@code Authorization: Bearer <token>}. */
    BEARER,
    /** Sends the token in the {@code X-API-Token} header. */
    API_TOKEN
}

File: csharp/AuthScheme.cs
namespace NewNanManager.Client;

/// <summary>
/// Credential header selected by the client.
/// </summary>
public enum AuthScheme
{
    /// <summary>Authorization: Bearer &lt;token&gt;.</summary>
    Bearer,

    /// <summary>X-API-Token: &lt;token&gt;.</summary>
    ApiToken,
}
